content/docs/command-reference/metrics/index.md

```diff
@@ -1,47 +1,104 @@
 # metrics
 
-A set of commands to add, manage, collect, and display project metrics:
-[add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove).
+A set of commands to manage, collect, and display _project metrics_:
+[show](/doc/command-reference/metrics/show), and
+[diff](/doc/command-reference/metrics/diff).
 
 ## Synopsis
 
 ```usage
 usage: dvc metrics [-h] [-q | -v]
-                   {show,add,modify,remove,diff} ...
+                   {show,diff} ...
 
 positional arguments:
   COMMAND
     show                Print metrics, with optional formatting.
-    add                 Mark a DVC-tracked file as a metric.
-    modify              Modify metric default formatting.
-    remove              Remove metric mark on a DVC-tracked file.
     diff                Show changes in metrics between commits
 ```
 
 ## Description
 
 In order to track metrics associated to machine learning experiments, DVC has
 the ability to mark a certain stage <abbr>outputs</abbr> as files containing
-metrics to track (see the `--metrics` option of `dvc run`). Metrics are
-project-specific floating-point values e.g. `AUC`, `ROC`, etc.
+metrics to track.
 
-Supported file formats: JSON. Metrics can be organized in a tree hierarchy in a
-JSON file. DVC addresses the metrics by the tree path.
+Metrics are project-specific floating-point or integer values e.g. AUC, ROC,
+_false positives_, etc. They can be defined with the `-m` (`--metrics`) and `-M`
+options of `dvc run`.
 
-DVC itself does not ascribe any specific meaning for these numbers. Usually
-these numbers are produced by the model training or model evaluation code and
-serve as a way to compare and pick the best performing experiment.
+### Types of metrics
 
-[Add](/doc/command-reference/metrics/add),
-[show](/doc/command-reference/metrics/show),
-[diff](/doc/command-reference/metrics/diff),
-[modify](/doc/command-reference/metrics/modify), and
-[remove](/doc/command-reference/metrics/remove) commands are available to set up
-and manage <abbr>DVC project</abbr> metrics.
+DVC has two concepts for metrics, that represent different results of machine
+learning training or data processing:
+
+1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
+   etc.
+2. `dvc plots` can be used to visualize **continuous metrics** such as AUC
```

Comment:
continuous - @dmpetrov are we okay using the term continuous after all, even though it's not very mathematically correct?

Reply:
FYI: The term "continuous metrics" is currently used 10 times.